lightning/src/sign/mod.rs

```diff
@@ -1903,6 +1901,10 @@ impl EntropySource for RandomBytes {
 	}
 }
 
+fn params_support_anchors(channel_params: &ChannelTransactionParameters) -> bool {
+	channel_params.channel_type_features.supports_anchors_zero_fee_htlc_tx()
+}
+
```

Comment:
Should we just make this a method on `ChannelTransactionParameters` rather than a standalone function?